migrations/045.do.measurement-lock.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE measurements ADD COLUMN lock NUMBER;
```

Comment:
Based on the test failures, I think the column type should be `INT` or `INTEGER`?